### ISSUE TITLE
Screen reader doesn't work on text within a view

### ISSUE THREAD

**Issue by dnadan2** [iamacup/react-native-markdown-display#172]
When creating a rule that returns a component to replace a particular tag it is possible to break the screen reader.

```
<View>
  <Text>Sample text that will not be read by a screen reader.</Text>
</View>
```

Even in a simple case like this the text is ignored by a screen reader (it is focused but is treated as a blank string.
Remove the view and this same text is read out correctly.

(Tested in an iOS simulator using the accessibility inspector tool)
